apps/settings/elements/firefox_sync.html

```diff
@@ -0,0 +1,92 @@
+<element name="fxsync" extends="section">
+  <template>
+
+    <gaia-header action="back" data-href="#root">
+      <h1 data-l10n-id="fxsync"></h1>
+    </gaia-header>
+
+    <div class="fxsync-logged-out" hidden>
+      <ul>
+        <li class="description">
+          <div class="fxsync-intro"></div>
+          <p data-l10n-id="fxsync-overview" id="fxsync-logged-out-description"></p>
+        </li>
+        <li>
+          <label>
```

Comment:
Is there a reason a label wraps a button? If not please remove it, it adds unnecessary clutter when using a screen reader.